argo-rollouts-demo/px-metrics.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: px-metrics
+  name: px-metrics
+  namespace: px-metrics
+spec:
+  selector:
+    matchLabels:
+      name: px-metrics
+  template:
+    metadata:
+      labels:
+        name: px-metrics
+        plane: control
+    spec:
+      containers:
+        - name: app
+          image: gcr.io/pixie-oss/pixie-dev/demo/argo-rollouts-demo:latest
+          env:
+          - name: PX_CLOUD_ADDR
+            value: withpixie.ai:443
```

Comment:
We can't do this, unfortunately. We may need to point to self hosted cloud instructions and tell them to fill this in.

Reply:
I will update this once the `px get cloudAddr` command lands.